<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>761</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QListView" name="deviceListView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>681</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startDiscoveryButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描设备</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serviceComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>571</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendDataLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>571</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendDataButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>310</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="receivedDataTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>571</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="stopDiscoveryButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>停止扫描</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>断开连接</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
